src/lib/divider/BUILD.bazel

```diff
@@ -7,12 +7,12 @@ ng_module(
   name = "divider",
   srcs = glob(["**/*.ts"], exclude=["**/*.spec.ts"]),
   module_name = "@angular/material/divider",
-  assets = [":divider_css"],
+  assets = [":divider_css"] + glob(["**/*.html"]),
```

Comment:
Do we need this here since divider has no html files (and likely never will)?

Reply:
It's better to have it for consistency with other components; it's just a no-op without any inputs